### ISSUE TITLE
Node.js Foundation Core Technical Committee (CTC) Meeting 2017-04-19

### ISSUE THREAD

**Issue by Trott** [nodejs/CTC#105]
## Time

**UTC Wed 19-Apr-2017 16:00 (04:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 19-Apr-2017 09:00 (09:00 AM) |
| US / Mountain | Wed 19-Apr-2017 10:00 (10:00 AM) |
| US / Central  | Wed 19-Apr-2017 11:00 (11:00 AM) |
| US / Eastern  | Wed 19-Apr-2017 12:00 (12:00 PM) |
| Amsterdam     | Wed 19-Apr-2017 18:00 (06:00 PM) |
| Moscow        | Wed 19-Apr-2017 19:00 (07:00 PM) |
| Chennai       | Wed 19-Apr-2017 21:30 (09:30 PM) |
| Tokyo         | Thu 20-Apr-2017 01:00 (01:00 AM) |
| Sydney        | Thu 20-Apr-2017 02:00 (02:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Core%20Technical%20Committee%20(CTC)+Meeting+2017-04-19&iso=20170419T16
* or http://www.wolframalpha.com/input/?i=4pm+UTC%2C+Apr+19%2C+2017+in+local+time

## Links

* **Minutes Google Doc**: <https://docs.google.com/document/d/1MGY5jZYpI2SMR4YKS3iKVG9LfQTv5Lez5-0LNpuYRek/edit>
* _Previous Minutes Google Doc: <https://docs.google.com/document/d/1nIXl2UMQs1n-yPencn53MYlfu3RnUltX3MfuaStPzx8/edit>_

## Agenda

Extracted from **ctc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* src: update --inspect hint text [#11207](https://github.com/nodejs/node/pull/11207)

## Invited

* Anna Henningsen @addaleax (CTC)
* Bradley Meck @bmeck (observer/GoDaddy/TC39)
* Ben Noordhuis @bnoordhuis (CTC)
* Сковорода Никита Андреевич @ChALkeR (CTC)
* Colin Ihrig @cjihrig (CTC)
* Evan Lucas @evanlucas (CTC)
* Franziska Hinkelmann @fhinkel (CTC)
* Jeremiah Senkpiel @Fishrock123 (CTC)
* Tracy Hinds @hackygolucky (observer/Node.js Foundation)
* Fedor Indutny @indutny (CTC)
* James M Snell @jasnell (CTC)
* Josh Gavant @joshgav (observer/Microsoft)
* Michael Dawson @mhdawson (CTC)
* Julien Gilli @misterdjules (CTC)
* Mikeal Rogers @mikeal (observer/Node.js Foundation)
* Brian White @mscdex (CTC)
* Myles Borins @MylesBorins (CTC)
* Ali Ijaz Sheikh @ofrobots (CTC)
* Jenn Turner @renrutnnej (observer/Node.js Foundation)
* Rod Vagg @rvagg (CTC)
* Shigeki Ohtsu @shigeki (CTC)
* Steven R Loomis @srl295 (observer/IBM/ICU)
* Michaël Zasso @targos (CTC)
* Sakthipriyan Vairamani @thefourtheye (CTC)
* Trevor Norris @trevnorris (CTC)
* Rich Trott @Trott (CTC)

## Notes

The agenda comes from issues labelled with `ctc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Uberconference; participants should have the link & numbers, contact me if you don't.

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the CTC that's not worth putting on as a separate agenda item, this is a good place for it.

**Comment by Trott** [comment]
In addition to the one item on the agenda, here are the 15 items currently labeled `ctc-review`. 

Would be nice to reduce this list, so if there's anything on here that doesn't need CTC input anymore, please by all means remove the label from that issue/PR.

* [module: remove require('.') with NODE_PATH compat](https://github.com/nodejs/node/pull/3384)
* [http: don't inherit from Object.prototype](https://github.com/nodejs/node/pull/6102)
* [errors, process: port internal/process errors to internal/errors](https://github.com/nodejs/node/pull/11294)
* [errors: assign error code to bootstrap_node created error](https://github.com/nodejs/node/pull/11298)
* [errors, child_process: migrate to using internal/errors](https://github.com/nodejs/node/pull/11300)
* [doc: add active, enroll and unenroll](https://github.com/nodejs/node/pull/11736)
* [async_hooks initial implementation](https://github.com/nodejs/node/pull/11883)
* [stream: move prefixed files into internal/streams.](https://github.com/nodejs/node/pull/11957)
* ~~[src, buffer: add --pending-deprecation flag](https://github.com/nodejs/node/pull/11968)~~ Landed
* [lib,src: exit process on unhandled promise rejection cleanup](https://github.com/nodejs/node/pull/12010)
* [fs: Optimizations 3: The Revenge](https://github.com/nodejs/node/pull/12105)
* [Emeritus status for CTC](https://github.com/nodejs/CTC/issues/92)
* [Who should have access to the security repo ?](https://github.com/nodejs/CTC/issues/96)
* [Why do people need/want to be org owners](https://github.com/nodejs/CTC/issues/97)
* [src: add require transform pipeline](https://github.com/nodejs/node/pull/12349)

**Comment by ChALkeR** [comment]
I am not sure if I will be present or not — I am not feeling well for the past few days, so it depends on how things will be on Wednesday for me.

**Comment by jasnell** [comment]
We should also make sure to discuss the planning around the delayed 8.0.0 release.

**Comment by addaleax** [comment]
I’ll try to show up but I’ll probably be ~30 min late, at this meeting as well as the next handful of meetings on this timeslot.

**Comment by targos** [comment]
I won't be present. I am on holiday this week.

**Comment by joshgav** [comment]
Background on the hint text agenda item in [this comment](https://github.com/nodejs/node/pull/11207#issuecomment-295265284). It would be a semver-major change after 8.0.0 (I think) hence tagged. With the decision to delay the release it's less urgent but your input would still be helpful. Thanks!

NB: I may miss today's meeting due to travel 😞 , so if you all prefer we can postpone discussion till next week per the delayed release plan.

cc @jkrems @eugeneo

**Comment by ChALkeR** [comment]
Sorry, I wasn't present at the meeting — I am not feeling well enough for voicechat.

**Comment by jasnell** [comment]
No worries @ChALkeR ... hope you feel better soon.

**Comment by mhdawson** [comment]
PR for minutes: https://github.com/nodejs/CTC/pull/108

**Comment by Trott** [comment]
I believe we close these issues once there's a PR for minutes, so I'm going to close this. If I'm wrong, of course, feel free to re-open...
